### ISSUE TITLE
The settings of continual test-time domain adaptation

### ISSUE THREAD

**Issue by Bojian-Chen** [mariodoebler/test-time-adaptation#10]
Hi, i am a phd student who's new to continual test-time domain adaptation.
In "Robust Mean Teacher for Continual and Gradual Test-Time Adaptation", the setting of continual test-time domain adaptation is that the model can be access to the data in source domain, but the origin setting in "continual test-time domain adaptation" i found is that the model only can be access to the data in target domian.
![image](https://github.com/mariodoebler/test-time-adaptation/assets/145563655/0040bf06-843e-450f-a7c2-008f13087354)
This is very confusing to me because in the setting where the model has access to the source domain, there is a significant improvement in the model's ability to overcome catastrophic forgetting and to adapt to the new domain. Can you please explain whether it is appropriate for your paper to be called Continuous Test-time Domain Adaptation.

**Comment by mariodoebler** [comment]
Hi @Bojian-Chen,

as described in our paper, we are looking at a broader range of practical applications: the common source-free setting (indicated by the source-free column in our result tables) and a setting where source data (or at least a small fraction of it, like 1%) may be available. If source data is available, source replay can be run as a simple extension. For warm-up, analogous to EATA (https://arxiv.org/abs/2204.02610), an out-of-distribution detection can be performed to collect some in-distribution data. The prototypes do not need access to source data, but can come together with an arbitrary model independent of its pre-training. A similar assumption is also made in the paper (https://www.ecva.net/papers/eccv_2022/papers_ECCV/papers/136930433.pdf). The test phase is source-free in the latter two cases. You can see the results of the individual settings in our component analysis in Table 4. I hope this helps!

Kind regards,
Mario

**Comment by Bojian-Chen** [comment]
Hi, Thank you for your prompt and patient response! 
I have a basic question:
Is the main difference between test-time adaptation and sourece-free domain adaptation in the testing phase, where test-time domain adaptation is trested after only one training epoch?
